### PR TITLE
Adding Sync units for Cryostat motors

### PR DESCRIPTION
**Pull request by jyotiphy**
## Description
Adding separate sync unit for CRIXS Cryostat motor 

## Motivation and Context
Had issue with these motors panel, so decided to create separate sync unit until the issue is resolved

## How Has This Been Tested?
Yes, it is tested and works fine. PLC was back in communication. 

## Where Has This Been Documented?
No, just here 

## Pre-merge checklist
- [x ] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to fixed versions and not ``Always Newest``
- [x] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT).xti [hunk 1]
```diff
@@ -50,6 +50,7 @@
 			<SyncUnit Name="VLS" NoDeleteIfUnused="true"/>
 			<SyncUnit Name="Catcher" NoDeleteIfUnused="true"/>
 			<SyncUnit Name="CRIX_VAC_PLC" NoDeleteIfUnused="true"/>
+			<SyncUnit Name="Cryostat" NoDeleteIfUnused="true"/>
```
Comment (ZLLentz) [inline lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT).xti:53]: Hi Jyoti, I don't see the rest of the changes typical of adding a sync unit. Usually this edits all the terminal files to add a line like this: https://github.com/pcdshub/lcls-plc-crixs-motion/blob/059a55f019bc246492ea808dd5d7d89b40d15853/lcls-plc-crixs-motion/_Config/IO/Device%201%20(EtherCAT)/Term%205%20(EK1200)/Term%20167%20(EK1122)/CRIX_VAC_INTF%20(EL6695-0002).xti#L48 which includes the sync unit reference for that io.

Are you sure this is working as intended?

Comment (ZLLentz) [reply lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT).xti:53]: I do vaguely remember there being a "good way" and a "bad way" to do sync units via the twincat ide and that one of them didn't seem to work correctly

Comment (jyotiphy) [reply lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT).xti:54]: You're right, didn't commit properly. Should be good now.

Comment (ZLLentz) [reply lcls-plc-crixs-motion/_Config/IO/Device 1 (EtherCAT).xti:54]: Looks much better now!

**Review by ZLLentz — APPROVED** [review]
Looks like the correct implementation to me + good decision overall

**Pull request merged**